inc/sainexthopgroup.h

```diff
@@ -133,6 +142,33 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_COUNTER_ID,
 
+    /**
+     * @brief Configured group size
+     *
+     * Maximum desired number of members. The real size should
+     * be queried from SAI_NEXT_HOP_GROUP_ATTR_ACTUAL_SIZE
```

Comment:
Did you mean SAI_NEXT_HOP_GROUP_ATTR_REAL_SIZE and not SAI_NEXT_HOP_GROUP_ATTR_ACTUAL_SIZE ?

Reply:
Yes, thanks